core/dbt/graph/selector_methods.py

```diff
@@ -356,7 +356,10 @@ def search(
             except AttributeError:
                 continue
             else:
-                if selector == value:
+                # if the config is a bool then check against the strings "true"/"false"
```

Comment:
Should we support all forms of yaml truthy/falsy (0/1, `"t`/`"f"`, `yes/no`)?

I'm fine with just `"true"` + `"false"` personally

Reply:
I'm in favour of just true/false and avoid the yaml-specify truthy/falsy rules which don't apply in Python/JSON/Jinja templates